src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -126,6 +136,10 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>) {
       // The work is now done. We don't need this anymore. This flags
       // to the system not to redo any work here.
       workInProgress.pendingProps = null;
+      workInProgress.updateQueue = null;
+      if (current) {
+        current.updateQueue = null;
```

Comment:
This looks scary. This whole work in progress tree can still be aborted so it is not safe to reset this yet. Wouldn't that drop work if that happens?